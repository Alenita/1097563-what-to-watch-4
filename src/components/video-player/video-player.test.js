import React from "react";
import VideoPlayer from "./video-player.jsx";
import renderer from "react-test-renderer";

const film = {
  id: 910,
  poster: `https://www.film.ru/sites/default/files/movies/posters/1610055-574057.jpg`,
  title: `Harry Potter - 1`,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`<VideoPlayer/> renders correctly`, () => {
  const {
    src,
    poster
  } = film;

  const tree = renderer.create(
      <VideoPlayer
        src={src}
        isPlaying={false}
        poster={poster}
        muted={true}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
