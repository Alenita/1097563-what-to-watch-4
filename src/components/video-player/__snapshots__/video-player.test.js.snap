// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer/> renders correctly 1`] = `
<video
  className="player__video"
  muted={true}
  poster="https://www.film.ru/sites/default/files/movies/posters/1610055-574057.jpg"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
/>
`;
